### ISSUE TITLE
Speed camera warns even if it is placed in the other direction 

### ISSUE THREAD

**Issue by dpriskorn** [organicmaps/organicmaps#6316]
**Describe the issue**
Organic maps warbs for speed cameras aimed in the opposite direction ecen if they have a direction tag.
Example that triggers when driving north on E4
https://www.openstreetmap.org/node/10092493470
![Screenshot_20231015-145715_OSMfocus Reborn.jpg](https://github.com/organicmaps/organicmaps/assets/68460690/7de721d0-179f-42d6-826d-9b30119a3bb6)

**Steps to reproduce**

**Expected behavior**

**Screenshots**
If applicable, add screenshots or screen recordings to help explain your problem.

**System information:**
 - Operating system and its version: [iOS 12, Android 10, Ubuntu 22, MacOS Big Sur, etc.]
 - Organic Maps version: [you can find it by tapping the button with the green OrganicMaps logo, bottom left of the main screen on Android, or the `?` button on iOS]
 - Device Model: [e.g. iPhone6, Samsung S22]

**Additional context**
Please add any other context and important details/notes/comments about the problem here.

**Comment by biodranik** [comment]
Can this camera shoot in any direction despite looking in one specific direction?

**Comment by dpriskorn** [comment]
> Can this camera shoot in any direction despite looking in one specific direction?

No, it's direction is completely fixed. I suppose the direction tag requires that to be the case or else it would not make sense

**Comment by muralito** [comment]
I suppose that he means that although it is fixed,  it could take the photo of the front or the back of the car?
In that location, are license plates mandatory front and rear?

**Comment by dpriskorn** [comment]
> I suppose that he means that although it is fixed,  it could take the photo of the front or the back of the car?
> In that location, are license plates mandatory front and rear?

Yes they are. But the enforcement is only one-way.
Maybe a front_only=yes tag would make this clear?

In Sweden every single speed camera is front_only

**Comment by muralito** [comment]
> Yes they are. But the enforcement is only one-way, only when facing the camera with the front. Maybe a front_only=yes tag would make this clear?

Maybe. But this would be not an OM issue, and the proposal process should be followed. https://wiki.openstreetmap.org/wiki/Proposal_process

Also, as you said that in all the country the cameras are "front only", OM (and other data consumers) could have a configuration like the right/left hand driving, so in some territories this front_only=yes is assumed by default.
